install/image-references

```diff
@@ -26,3 +26,11 @@ spec:
     from:
       kind: DockerImage
       name: quay.io/openshift/origin-azure-machine-controllers:v4.0.0
+ - name: baremetal-operator
+    from:
+      kind: DockerImage
+      name: quay.io/metal3-io/baremetal-operator:latest
+ - name: baremetal-provider
+    from:
+      kind: DockerImage
+      name: quay.io/repository/metal3-io/ironic:/latest
```

Comment:
We'll need to get the ironic image in openshift next.  That's going to block this PR

Reply:
Yes, will keep this one as WIP until we have the ironic image in openshift and make the necessary changes here.

Reply:
The blocker here has been resolved. [https://github.com/openshift/machine-api-operator/pull/367]